README.md

```diff
@@ -136,8 +145,8 @@ remember: for support, questions, or feature requests contact: deeptools@googleg
 [bamCoverage]: https://github.com/fidelram/deepTools/wiki/Normalizations#wiki-bamCoverage
 [bamCompare]: https://github.com/fidelram/deepTools/wiki/Normalizations#wiki-bamCompare
 [computeMatrix]: https://github.com/fidelram/deepTools/wiki/Visualizations
-[heatmapper]: https://github.com/fidelram/deepTools/wiki/Visualizations
-[profiler]: https://github.com/fidelram/deepTools/wiki/Visualizations
+[plotHeatmap]: https://github.com/fidelram/deepTools/wiki/Visualizations
+[plotProfile]: https://github.com/fidelram/deepTools/wiki/Visualizations
 [FAQ]: https://github.com/fidelram/deepTools/wiki/FAQ
```

Comment:
No, we'll keep it around forever for <2.0 releases. We need to add a link at the top of each page to read the docs, though (it's on my to do list).